app/cryptography/jwt_encoder.py

```diff
@@ -4,26 +4,22 @@
 from cryptography.hazmat.primitives.ciphers import algorithms
 from cryptography.hazmat.primitives.ciphers import modes
 
-from app import settings
 from app.submitter.encrypter import Encrypter
 
 KID = 'EDCRRM'
 
 
 class Encoder(Encrypter):
-    def __init__(self):
-        # pylint: disable=maybe-no-member
-        # key variables are dynamically assigned
-        if settings.EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY is None:
+    def __init__(self, private_key, private_key_password, public_key):
+
+        if private_key is None:
             raise Exception("EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY is None (check EQ_DEV_MODE?)")
-        if settings.EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY_PASSWORD is None:
+        if private_key_password is None:
             raise Exception("EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY_PASSWORD is None (check EQ_DEV_MODE?)")
-        if settings.EQ_USER_AUTHENTICATION_SR_PUBLIC_KEY is None:
+        if public_key is None:
             raise Exception("EQ_USER_AUTHENTICATION_SR_PUBLIC_KEY is None (check EQ_DEV_MODE?)")
```

Comment:
Is there value in adding some stronger validation here as well? We could push the validation into encrypter and then this init method could be removed altogether from here